lib/getLatestRelease.js

```diff
@@ -75,13 +95,14 @@ const prepAssets = assets => {
       // Omit OS X ZIP files (used for update purposes)
       // Omit electron packages
       return !(
+        (includes(item.name, '.zip') && includes(item.name, 'mac')) ||
         (includes(item.name, '.zip') && includes(item.name, 'darwin')) ||
         includes(item.name, 'electron')
       );
     })
     .map(item => {
       return {
-        text: prettifyFileName(item.name),
+        text: packagePrettyName(item.name),
         href: item.browser_download_url,
         os: getOS(item.name),
         arch: getArch(item.name)
```

Comment:
Just to be on the safe side, it might be worth lower-casing `item.name` everywhere where it's used?